ibm-mas/gitops
# [minor] Add sanity and verify tests for Maximo IT

Take changes from https://github.com/ibm-mas/gitops/pull/233 and https://github.com/ibm-mas/gitops/pull/242 and incorporate into gitops

Added Sanity (invasive) Testcases for Maximo IT.

To verify Maximo IT version detail in system info
Test to verify Self Serve is installed
Test to verify Service View is installed
To test create a incident and validate its creation
To test create a Asset and validate its creation
To test create a ticket template and validate its creation
To test create a Solution and validate its creation
To test create a problem and validate its creation
To test create a change request and validate its creation
To test create a Service Request and validate its creation
To test create a Bulletin Message and validate its creation
Test to get all incidents
Test fetching all ticket templates
Test fetching all problems
Test fetching all assets
Test fetching all service request
Test fetching all solutions
Test fetching all change request
Test fetching all Bulletin Board Message
To test search incident by incident description
To test search Service Request by its summary
To test search Problem by its summary
Create a new service and validate its creation
Test to check error message when editing service name
To test error message displayed when updating service with blank Service Name
Test to update description of service
Test to update service status as Draft
Test to update summary and long description of incident
Test to update summary and details of problem
Test to update summary and detail of SR
Test to update summary and detail of Change Request
Test to update summary and details of solution
Test to fetch all services
Test to fetch all services based on filter criteria service name
Test to fetch all services based on filter criteria service description
Test to fetch all services based on filter criteria status Active
Test to fetch all services based on filter criteria status Draft
Fetch all services based on filter criteria service provision Inhouse
Fetch all services based on filter criteria owned by
Create a Ticket and validate its creation
Get the list of tickets displayed in My Items View More
Request Update on the ticket in MyItems in self serve
Adding comment in Ticket from My Items

Added 3 verify (non invasive) test cases.
#TC#1 Test to verify Maximo IT is installed
#TC#2 Test to verify Self Serve is installed
#TC#3 Test to verify Service View is installed
